### PR TITLE
Add write to disk options for build command

### LINKED ISSUE #4826: Add write-to-disk watch mode for kolibri tools to allow single origin dev testing.
### Observed behavior

This appears to be due to a browserstack limitation: https://www.browserstack.com/question/758

As a workaround, http://kolibribeta.learningequality.org works fine

Browserstack reports:

> image

We see: 

> image

### Expected behavior

Kolibri should load

### User-facing consequences

unable to use on iOS

### Errors and logs

from Browserstack console: 

> image

### Steps to reproduce

load kolibri on safari browserstack:

> image

Note that you need to enter your local network IP address, _not_ localhost on iOS. Or just use http://kolibribeta.learningequality.org

### Context

#4589 looks related

k0.12

browserstack device info:

> image

### PR DESCRIPTION
**Pull request by thesujai**
## Summary

This exposes `--write-to-disk` flag in build that writes file to disk instead of webpack

## References

Fixes #4826 

## Reviewer guidance

In https://github.com/learningequality/kolibri/blob/develop/package.json#L15 Should there be a `prod` argument?

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: packages/kolibri-tools/lib/cli.js [hunk 1]
```diff
@@ -320,7 +321,19 @@ program
...
     }
-    runWebpackBuild(mode, bundleData, mode === modes.DEV, options);
+
+    if (options.writeToDisk && mode === modes.DEV) {
+      cliLogging.warn(
+        'Enabling write-to-disk mode may fill up your developer machine with lots of different built files if frequent changes are made.'
+      );
+      runWebpackBuild(mode, bundleData, false, {
+        ...options,
+        cache: false,
```
Comment (rtibbles) [inline packages/kolibri-tools/lib/cli.js]: Is there a reason not to utilize the webpack cache here? It should help with build performance either way.

Also, the use of hot is limited to the devserver being active, which we are passing false for, so I think this `if` could do the warn, then we could update the runWebpackBuild to simply be:

```
runWebpackBuild(mode, bundleData, !options.writeToDisk && mode === modes.DEV, options);
```

and not have to invoke it separately in the if statement.

**Review by rtibbles — CHANGES_REQUESTED** [review]
I think the diff can be a bit smaller, and no need to disable the cache in this case that I can see.

File: packages/kolibri-tools/lib/cli.js (hunk 1 above)
Comment (thesujai) [reply packages/kolibri-tools/lib/cli.js]: yes, I thought cache will fill the dev's system

Comment (thesujai) [reply packages/kolibri-tools/lib/cli.js]: So yes, if hot and cache are not being modified then i will run the webpack build with your suggestions. Thank You

Comment (rtibbles) [reply packages/kolibri-tools/lib/cli.js]: The cache is better managed than the writing built files to disk, so I wouldn't worry too much!

Comment (thesujai) [reply packages/kolibri-tools/lib/cli.js]: Done!! Thank You

**Review by rtibbles — APPROVED** [review]
Excellent, thank you!

**Pull request merged**